partnercenter/azext_partnercenter/operations/marketplace_offer_plan/_params.py

```diff
@@ -4,10 +4,15 @@
 # --------------------------------------------------------------------------------------------
 # pylint: disable=line-too-long
 
-from azure.cli.core.commands.parameters import (get_enum_type, get_three_state_flag)
+from azure.cli.core.commands.parameters import (get_enum_type,
+                                                get_three_state_flag)
 
 
 def load_arguments(commands_loader, _):
-    pass
-    # with commands_loader.argument_context('partnercenter marketplace offer') as c:
-    #     c.argument('arg', options_list=['--arg', '-a'], help='The argument help')
\ No newline at end of file
+    with commands_loader.argument_context('partnercenter marketplace offer plan') as c:
+        c.argument('offer_resource_id', options_list=['--offer-resource-id'], help='The Offer Resource ID of the Offer for the Plan in the partner center.')
```

Comment:
using convention "parent resource name" + "resource-id" so that we can support both a resource id in the CLI and differentiate it from "--id" of the current resource, such as "plan"